src/helper.ts

```diff
@@ -1,5 +1,6 @@
 // This file is for Deno to import helpers from `hono/helper.ts`.
+export * from './helper/adapter'
 export * from './helper/cookie'
 export * from './helper/html'
-export * from './helper/adapter'
+export * from './helper/stream'
```

Comment:
`./helper/streaming/index.ts` should be better.

Reply:
fixed